File: src/main/java/com/bookmakerApp/view/BookmakerClientApp/BookmakerClientApp/Data/Model/FootballEventModelDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BookmakerClientApp.Data.Model
{
    public class FootballEventModelDto
    {
        private String footballMatchType { get; set; }
        private String homeTeamName { get; set; }
        private String homeTeamCountry { get; set; }
        private int homeTeamGoals { get; set; }
        private String visitingTeamName { get; set; }
        private String visitingTeamCountry { get; set; }
        private int visitingTeamGoals { get; set; }
    }
}

File: src/main/java/com/bookmakerApp/facade/dtos/event/FootballEventModelDto.java
package com.bookmakerApp.facade.dtos.event;

import com.bookmakerApp.facade.dtos.event.DefaultEventModelDto;
import lombok.Getter;
import lombok.experimental.SuperBuilder;

@Getter
@SuperBuilder
public class FootballEventModelDto extends DefaultEventModelDto {
    private String footballMatchType;
    private String homeTeamName;
    private String homeTeamCountry;
    private int homeTeamGoals;
    private String visitingTeamName;
    private String visitingTeamCountry;
    private int visitingTeamGoals;
    private int numberOfPages;
}
